tests/e2e/fixtures/html/linenumberStart.html

```diff
@@ -1,10 +1,10 @@
 <template>
-    <span :title="_xn('title', '%{n} apple', '%{n} apples', n, {n:''+n})">
+    <span :title="_xn('title', '%{n} apple', '%{n} apples', n, {n:''+n})" data-name="__('whatever')">
         {{ _n( '%{n} apple', '%{n} apples', n, {n:''+n}) }}
     </span>
     <div>
         {{ __('msg 1') }}
         <br>
         {{ __('msg 2') }}
```

Comment:
I just now realized what this was and why you needed the embeddedJs extractor with a `*` selector. I do not officially support the (ab)use of the embeddedJs extractor to process template syntax like this because it isn't always valid JavaScript. Of course people are free to use my package however they like, but I don't want this kind of example in my tests.